pkg/packageinstall/packageinstall_test.go

```diff
@@ -454,3 +454,100 @@ func Test_PlaceHolderSecretCreated_WhenPackageInstallUpdated(t *testing.T) {
 
 	assert.Equal(t, "instl-pkg-fetch-0", app.Spec.Fetch[0].ImgpkgBundle.SecretRef.Name)
 }
+
+func Test_CreatePackage_RevokedStatus(t *testing.T) {
```

Comment:
I was expecting to see a test that shows that if you have versions 1.0.0 and 1.1.0 and you do the version selection logic, it returns 1.1.0.    but then if you revoke 1.1.0 and run the same version selection function again,  it returns 1.0.0. 

i think similarly you could cover the "don't upgrade to a revoked version" by testing that same version selection function and showing that if you then add 1.2.0  package it'll return that one (skipping the revoked package).

Reply:
This PR doesn't cover any upgrading patterns, it just allows and then shows the revoked status. The next iteration is the one where this fun comes into play. If I read your comment correctly!